### PR TITLE
Initial draft of FAQs for speakers and hosts

### PR DESCRIPTION
**Pull request by ZauberNerd**

### CODE REVIEW

**Review by learosema — APPROVED** [review]

File: hosts.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## What is the typical duration of a meetup?
+
+Usually, attendees start arriving at 7pm, while the actual meetup starts at 7:30pm and ends before 10pm.
+
+## Can we address the attendees?
+
+You will have up to 5 minutes at the beginning of the meetup to present your organisation. Additionally you are welcome to socialise with our attendees during breaks.
+
+_Current employees of hosts are not selected to give talks on the months of their employer's sponsorship._
```
Comment (herschel666) [inline hosts.md]: I'd prepend a ⚠️ here.

File: speakers.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Examples of topics that may not get accepted:
+- Why x framework is better than y framework.
+- Startup or company pitches.
+
+ We've found that the best talks tell a story, we encourage you to get creative!
+
+## When are speakers chosen?
+We usually select speakers two weeks before an event.
+
+_Current employees of hosts are not selected to speak on the months of their employer's sponsorship._
```
Comment (herschel666) [inline speakers.md]: I'd prepend a ⚠️ here.

File: speakers.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+## I have more questions about submitting talks, what should I do?
+Please feel free to ping us on twitter [@hhjs](https://twitter.com/hhjs) or join us in our [WebHH Slack](https://web-hh-slackin.herokuapp.com/) group [#hhjs](https://web-hh.slack.com/messages/C0KESM96C)
```
Comment (herschel666) [inline speakers.md]: I'd not include twitter here, if we don't have a responsible person for that channel. Furthermore we want ppl to join the Slack organization, so having only this channel (for now) is imho acceptable.

File: speakers.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+# Speaking at HH.js
+We are excited to have you as speaker! Here is few things to keep in mind when you are making your talk. If you have any questions or want to discuss your talk with organizers, please feel free to ping us!
```
Comment (herschel666) [inline speakers.md]:
```diff
- Here is few things to keep in mind when you are making your talk.
+ Here are a few things to keep in mind when giving your talk.
```

```diff
- …, please feel free to ping us!
+ …, please feel free to ping us via <…>!
```

File: speakers.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+## How long is my talk?
+Talks are expected to be 20 minutes long, lightning talks should be limited to 5 minutes.
+
+## Speaker Checklist
+### Slides
+If your are showing code or your terminal, please keep your font size large and contrast high!
```
Comment (herschel666) [inline speakers.md]:
```diff
- …and contrast high!
+ …and the contrast high!
```

**Review by herschel666 — CHANGES_REQUESTED** [review]

File: speakers.md (hunk 4 above)
Comment (ZauberNerd) [reply speakers.md]: I've changed these sentences a bit, please take another look.

As for your second suggestion: What should we put there? The `#hhjs` slack channel? Or tell them to ping us via private messages on slack?

Comment (herschel666) [reply speakers.md]: I would be fine with Slack.

Comment (ZauberNerd) [reply speakers.md]: I linked this with the other paragraph that contains all the links to twitter, slack, etc.

File: speakers.md (hunk 3 above)
Comment (dmbch) [reply speakers.md]: For the time being, I will answer all Twitter DMs, so in my opinion, we can keep it in.

**Review by dmbch — APPROVED** [review]

**Pull request merged**